modules/rosa-hcp-firewall-prerequisites.adoc

```diff
@@ -106,30 +90,29 @@ endif::rosa-classic-sts[]
 Managed clusters require enabling telemetry to allow Red{nbsp}Hat to react more quickly to problems, better support the customers, and better understand how product upgrades impact clusters.
 For more information about how remote health monitoring data is used by Red{nbsp}Hat, see _About remote health monitoring_ in the _Additional resources_ section.
 
-. Allowlist the following Amazon Web Services (AWS) API URls:
-+
+== Domains for Amazon Web Services (AWS) APIs
 [cols="6,1,6",options="header"]
 |===
 |Domain | Port | Function
 
 |`sts.<aws_region>.amazonaws.com` ^[1]^
 |443
 |Required. Used to access the AWS Secure Token Service (STS) regional endpoint. Ensure that you replace `<aws-region>` with the region that your cluster is deployed in.
-
-|`sts.amazonaws.com` ^[2]^
-|443
-|See footnote. Used to access the AWS Secure Token Service (STS) global endpoint.
 |===
 +
```

Comment:
This plus shows up in the live build.

![image](https://github.com/user-attachments/assets/5f3efcb8-b999-4ca9-b902-d728c8a87cd9)